modules/social_features/social_comment/social_comment.install

```diff
@@ -60,3 +60,19 @@ function social_comment_install() {
 function social_comment_update_last_removed() : int {
   return 11202;
 }
+
+/**
+ * Add profile's First Name and Last Name in combined filter to view Comments.
+ *
+ * Change search operator.
+ */
+function social_comment_update_12001(): string {
```

Comment:
@VBaldych Since we want to release it in version 12.2.x, better to rename hook update to `_12201`